GenericSerialDeviceRemoteControl/Resources/UI/GenericSerialDeviceRemoteControl.ui

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>Form</class>
+ <widget class="QWidget" name="Form">
+   <layout class="QVBoxLayout" name="verticalLayout_2">
+    <item>
+     <widget class="ctkCollapsibleButton" name="plusParametersCollapsibleButton">
+      <property name="text">
+       <string>Plus parameters</string>
+      </property>
+      <property name="collapsed">
+       <bool>false</bool>
+      </property>
+       <layout class="QFormLayout" name="formLayout">
+        <item row="0" column="1">
+         <widget class="qMRMLNodeComboBox" name="connectorNodeSelector"/>
+        </item>
+        <item row="1" column="1">
+         <widget class="QComboBox" name="deviceIdComboBox"/>
+        </item>
+        <item row="0" column="0">
+         <widget class="QLabel" name="label">
+          <property name="text">
+           <string>Connector node:</string>
+          </property>
+         </widget>
+        </item>
+        <item row="1" column="0">
+         <widget class="QLabel" name="label_2">
+          <property name="text">
+           <string>Device ID:</string>
+          </property>
+         </widget>
+        </item>
+       </layout>
+     </widget>
+    </item>
+    <item>
+     <widget class="ctkCollapsibleButton" name="sendCommandCollapsibleButton">
+      <property name="text">
+       <string>Send Generic Command</string>
```

Comment:
How about just "Send Command"? Technically the command itself can be for something very specific.